src/main/scala/com/fulcrumgenomics/fastq/DemuxFastqs.scala

```diff
@@ -375,6 +385,10 @@ class DemuxFastqs
       }
     }
 
+    // validate the quality threshold provided
+    require(qualityThreshold >= 0 && qualityThreshold <= 40,
+      s"The quality threshold ($qualityThreshold) is not within the expected range (0, 40)")
```

Comment:
Why 40?  I would argue there's no need to restrict this on the upper end, or even at all - what harm does it do if someone specifies `-10` or `60`?